superset-frontend/src/dashboard/components/nativeFilters/FilterCard/ScopeRow.tsx

```diff
@@ -47,17 +47,16 @@ export const ScopeRow = React.memo(({ filter }: FilterCardRowProps) => {
     [elementsTruncated, scope],
   );
 
-  if (!Array.isArray(scope) || scope.length === 0) {
-    return null;
-  }
   return (
     <Row>
       <RowLabel>{t('Scope')}</RowLabel>
       <TooltipWithTruncation title={tooltipText}>
         <RowValue ref={scopeRef}>
-          {scope.map((element, index) => (
-            <span>{index === 0 ? element : `, ${element}`}</span>
-          ))}
+          {scope
```

Comment:
This change was added in https://github.com/apache/superset/pull/18793 (in review), copying it here in order to write a test for it before #18793 is merged